app/waf/test/acceptance/application.controller.acceptance.ts

```diff
@@ -218,13 +218,68 @@ describe('ApplicationController', () => {
       .expect(204);
   });
 
+  it(
+    'post ' +
+      prefix +
+      '/applications/{id}/deploy: deploy an unspported action.',
```

Comment:
What is "unspported action"? The AS3 action is made by our code. Why it will become "unspported?

Reply:
correct the comments with "deploy an application without partition".